apps/portal/src/components/layout/HeaderActions.astro

```diff
@@ -11,11 +11,7 @@ import ThemeSelect from "./ThemeSelect.astro";
   >
     Blog
   </Button>
-  <Button
-    variant="ghost"
-    size="icon"
-    className="text-cn-foreground-1 p-0 hover:bg-transparent hover:text-cn-foreground-primary"
```

Comment:
Shall we remove `className` prop from components? If we need any other variant, it should be added via design system. Thoughts? 

cc: @abhinavrastogi-harness @knagurski

Reply:
hmm we do need to use classNames for layout purposes sometimes.. like adding margins, align etc.
Not having a `className` prop forces devs to wrap it in unnecessary divs.

Reply:
allowing `className` will add risk of altering design system's padding, height, etc,.. and it could make maintainability a complex task.